### PR TITLE
WIP: python3Packages.bokeh: 1.4.0 -> 2.0.1

### PR DESCRIPTION
**Pull request by FRidh**
Tests still don't function. Somehow fixtures cannot be found causing massive failures...

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by FRidh** [comment]
note 2.0.2 has been tagged

File: pkgs/development/python-modules/bokeh/default.nix [hunk 1]
```diff
@@ -55,13 +95,14 @@ buildPythonPackage rec {
...
     packaging
+    typing-extensions
   ]
   ++ lib.optionals ( isPy27 ) [
     futures
   ];
 
   checkPhase = ''
-    ${python.interpreter} -m unittest discover -s bokeh/tests
+    pytest tests
```
Comment (jonringer) [inline pkgs/development/python-modules/bokeh/default.nix:105]: i see you declared `disabledTests` did you mean to use `pytestCheckHook`?

Comment (FRidh) [reply pkgs/development/python-modules/bokeh/default.nix:105]: yea I was initially but since I needed to debug more that wasn't helpful

**Comment by FRidh** [comment]
pushed 2.0.2 in 491b4dafe6228bf2399bb0806157a50da8e68e5c (note incorrect commit message)

**Pull request closed without merging**